src/Utilities/FileSystem.cpp

```diff
@@ -263,4 +264,20 @@ void rm(const std::string& path, bool recursive) {
     // LCOV_EXCL_STOP
   }
 }
+
+std::vector<std::string> glob(const std::string& pattern) {
+  glob_t buffer;
+  const int return_value =
+      ::glob(pattern.c_str(), GLOB_TILDE, nullptr, &buffer);
+  if (return_value != 0) {
+    throw std::runtime_error("Unable to resolve glob '" + pattern +
```

Comment:
Why not use `ERROR`? That will hopefully `throw` soon. If it's necessary I can see if I can do the conversion to exceptions

Reply:
PR for ABORT/ERROR throwing is almost ready

Reply:
ok switched to `ERROR`.

Once `ERROR` throws, shall we use exceptions directly more?

Reply:
Yea, the plan is to only use exceptions everywhere. Then the Algorithm will catch exceptions, print them, then call `parallel::abort`. In the future we can extend this to flush data to disk before terminating the simulation so that we have all possible data we might need. We could even trigger a full write to disk of the entire computational domain to aid in debugging.